vaadin-spring-tests/test-spring-security-fusion/frontend/views/main-view.ts

```diff
@@ -77,7 +77,7 @@ export class MainView extends Layout {
 
   private logout() {
     logout();
-    Router.go("");
```

Comment:
Why doesn't `""` work with a context path?

Reply:
A bug in vaadin-router? with `Router.go("")`, it goes to the root URL without the context path.

Reply:
Made a ticket https://github.com/vaadin/router/issues/636, let's merge this PR first, I have another PR https://github.com/vaadin/spring/pull/839 depends on this one.